modules/admin_manual/pages/faq/index.adoc

```diff
@@ -1,30 +1,65 @@
 = Frequently Asked Questions
 :toc: right
 
+[[cannot-see-new-ldap-accounts]]
+== I cannot see new LDAP accounts
+
+Set up a new cronjob for your Apache user to run occ with the user:sync command regularly.
+
+Example for Ubuntu:
+
+....
+crontab -u www-data -e
+....
+
+WARNING: Replace "www-data" with the correct Apache user for your Linux distribution.
+
+Insert
+
+....
+*/6 * * * * /usr/bin/php /var/www/owncloud/occ user:sync -n -m disable -r "OCA\User_LDAP\User_Proxy"
+....
+
+Save and exit. The command will now be executed every ten minutes.
+
 [[i-want-to-upgrade-from-community-version-to-enterprise-version.-what-are-the-changes]]
```

Comment:
No need to add an explicit id if the text matches the text in the header. One exactly the same will be automatically generated.

Reply:
I see. Honestly, I just took what was already there and modified it. Will try to keep that in mind for the upcoming PRs.